packages/cli/src/main/kotlin/elide/tool/cli/ElideTool.kt

```diff
@@ -228,6 +228,16 @@ import elide.tool.io.RuntimeWorkdirManager
   )
   internal var srcfile: String? = null
 
+  /** Script file arguments. */
+  @Parameters(
+    index = "1",
+    description = ["Arguments to pass"],
+    scope = ScopeType.INHERIT,
+    arity = "0..*",
+    paramLabel = "ARG",
+  )
+  internal var args: List<String>? = null
```

Comment:
Args can be passed to the main entrypoint (like `elide some-script.py args here`)
Or to the shell entrypoint (like `elide run some-script.js args here`)